packages/protocol/src/data/inscription.ts

```diff
@@ -762,6 +762,7 @@ export interface ScriptingDataArgs {
 }
 export interface TypeSearchRequest {
   context: InscriptionContext;
+  limit: number;
```

Comment:
so the limit can no be defined by the client?

Reply:
Yes, the client must specify a limit, otherwise no data will be returned.